storage.c

```diff
@@ -1090,10 +1169,10 @@ struct extstore_conf_file *storage_conf_parse(char *arg, unsigned int page_size)
     }
 
     // TODO: disabling until compact algorithm is improved.
-    if (cf->free_bucket != PAGE_BUCKET_DEFAULT) {
+    /*if (cf->free_bucket != PAGE_BUCKET_DEFAULT) {
         fprintf(stderr, "ext_path only presently supports the default bucket\n");
         goto error;
-    }
+    }*/
```

Comment:
This could just be removed instead of commenting out since these lines are stored in git.